Add export_empty_as and  export_sort as parameters

maurovc opened the pull request:
Hey! I created a fork to add two new parameters: `export_empty_as` and  `export_sort`.

I've seen that you have a PR to allow additional params by sending a hash but this PR adds defaults and checks to these params so I think that it's still of value.

Let me know what you think!

Thanks 🙃

Commit:
Added new API parameters

Commit:
Added parameters to FastlaneCore config

Commit:
Updated export_sort description

Commit:
Added new parameters

Commit:
Type tags parameter as Array

Commit:
Add file_strategy

Commit:
Implement merge file-strategy

Commit:
Merge pull request #1 from Glovo/file-strategy

File strategy

Commit:
v2.0.7

Commit:
Update to use a hash

Commit:
Add update file strategy

Commit:
Merge pull request #2 from Glovo/update-fyle-strategy

Add update to the file strategy list

Commit:
v2.0.9

Commit:
Fix elsif

Commit:
v2.0.10

Commit:
Fix strategy broken for non .strings files

Commit:
v2.0.11

andrew54068 commented:
if anyone wants to remove empty string for localiaztion strings this might help.
```desc "remove empty value string in localized string file."
private_lane :removeEmptyString do
  Dir.chdir("..") do
    sh("find ./yourpath/* -name '*.strings' | xargs sed -i '' '/\"\";/d'")
  end
end
```

dcacenabes committed:
Add API V2 support

dcacenabes committed:
Rename parameters

dcacenabes committed:
Merge pull request #3 from Glovo/tech/migrate-to-api-v2

Add API V2 support

dcacenabes committed:
v2.0.12

fabfelici committed:
Added escape_percent parameter

fabfelici committed:
Added escape_percent parameter support

dcacenabes committed:
Merge pull request #4 from Glovo/escape_percent

Added escape_percent parameter support

dcacenabes committed:
v2.0.13

dcacenabes committed:
Fix typo in error handling

dcacenabes committed:
Merge pull request #5 from Glovo/fix/error-handling

Fix typo in error handling

dcacenabes committed:
v2.0.14

dcacenabes committed:
Fix tags request

Fix malformed request error

dcacenabes committed:
Merge pull request #6 from Glovo/fix/tags_request

Fix tags request

dcacenabes committed:
v2.0.15

dcacenabes committed:
Update domain for the lokalise API

The domain .co has been shutdown. We must now do requests to the .com domain.

dcacenabes committed:
Merge pull request #7 from Glovo/fix-api

Update domain for the lokalise API

dcacenabes committed:
v2.0.16

dcacenabes committed:
Fix empty space indentation

dcacenabes committed:
Fix file processing

Ensure that we always remove newline character when reading the file and add new lines manually when writing to the file

dcacenabes committed:
Merge pull request #8 from Glovo/fix-newline-issue

[MOB] Fix adding 2 strings into the same line

dcacenabes committed:
v2.0.17

Commit:
Update lokalise_action to run only on existing files

We need to apply the given operation based on the file strategy only if the file exists. This is needed because we could not have all the files that existing in the localization repo in the same directory as we were used to have.

angelstoone committed:
Apply CR suggestion

Co-authored-by: David Cacenabes <david.cacenabes@gmail.com>

angelstoone committed:
Merge pull request #9 from Glovo/feature/CCE-995/apply_only_if_file_exists

[CCE-995] Apply operation related to file strategy only if file exists
